BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -226,20 +228,22 @@ def register(task)
       # theres already one. Only applies to Bourreaux (as it would make no
       # sense on the portal).
       return if Rails.root.to_s =~ /BrainPortal\z/
-
       # Create a ToolConfig iff
       #   (1) the Bourreau has a docker executable and
       #   (2) the descriptor specifies a docker image
-      return if docker_image.nil? || !resource.docker_present
+      return if container_image.nil? || container_engine.nil?
```

Comment:
Why use `.nil?` This will fail if the value is an empty string, because `"".nil?` returns false